### DISCUSSION TITLE
Need help for this error Class App\Models\Settings\EmailSettings has no unserializer

### DISCUSSION THREAD

**Discussion by mohammadsyuhada** [spatie/laravel-settings#111] [Q&A] [answered]
Hi, this issue might be related to #105. Currently im using this package with PHP8.0. Just upgraded to latest 2.3.1 version, and got this error.
 
Im loading this EmailSettings in AppServicePovider like below. If using 2.2.0, it works. 
```
config([
    'mail.default' => app(EmailSettings::class)->use_mailgun ? 'mailgun' : 'log',
    'services.mailgun.domain' => app(EmailSettings::class)->mailgun_domain,
    'services.mailgun.secret' => app(EmailSettings::class)->mailgun_secret,
    'mail.from.address' => app(EmailSettings::class)->general_email_sender,
    'mail.from.name' => app(EmailSettings::class)->general_email_sender_name
]);
```

```
Class App\Models\Settings\EmailSettings has no unserializer
 at vendor/spatie/laravel-settings/src/SettingsCache.php:50
     46▕         }
     47▕ 
     48▕         $serialized = Cache::store($this->store)->get($this->resolveCacheKey($settingsClass));
     49▕ 
  ➜  50▕         $settings = unserialize($serialized);
     51▕ 
     52▕         if (! $settings instanceof Settings) {
     53▕             throw new CouldNotUnserializeSettings();
     54▕         }
```

**Comment by rubenvanassche** [comment, marked as answer]
Hi @kotakSempit,

The `Serializable ` interface was deprecated(https://php.watch/versions/8.1/serializable-deprecated) so that's why we don't use it anymore. Normally `unserialize()` should work fine without the interface and the `unserialize()` method since it will use the `__unserialize()` method.

Just ran the tests on PHP 7.4 without any problems and we're using `unserialize` over there.

Could you try cleaning your cache and try again.

**Reply by mohammadsyuhada** [reply]
Tried this. And it works! Totally forgot about the cache. Thank you for the help and for this package in general.
